### PR TITLE
Raptorborg resprite and additional raptorborg types

### PR DESCRIPTION
**Pull request by SpaceMan-Jim**
Oh no, they figured out how to run!

Gives all current and newly added raptorborg silicons movement animations.
New module sprites added: ERT Raptor, Syndicate Raptors for their respective type, Gravekeeper and Lost/Stray raptor.

Other additions: Various sprite layers for the raptors like taser/laser/gun/disabler sprites, shield sprites and even a speed/mobility module sprite for the ERT/Combat raptor. One new energy weapon sound effect for the syndicate borg suppressor/dakka laser (from what I could tell, it will be the only energy gun to be changed... and yes, that is the Terminator plasma gun sound effect.)

The gang

https://github.com/VOREStation/VOREStation/assets/127170206/786dc321-9efb-4018-8120-8f1418c016e8

Now this is probably my first big (ish) PR, so do not be afraid to say if it could be more streamlined/clean with the code changes.

### CODE REVIEW

**Comment by Heroman3003** [comment]
It seems like PR has been made on an outdated branch and has conflict. Need to resolve those.

File: code/modules/mob/living/silicon/robot/sprites/medical.dm [hunk 1]
```diff
@@ -251,7 +251,7 @@
 		SP.attack_verb = list("batted", "pawed", "bopped", "whapped")
 
 /datum/robot_sprite/dogborg/tall/surgical/raptor
-	name = "Raptor V-4"
+	name = "Raptor Surgical"
```
Comment (Heroman3003) [inline code/modules/mob/living/silicon/robot/sprites/medical.dm:254]: Im unsure about changing the names considering the modules are separate and there is no overlap, and having same name for same 'sprite type' is good for recognizability. Same applies to Crisis and Syndie modules below.

Comment (SpaceMan-Jim) [reply code/modules/mob/living/silicon/robot/sprites/medical.dm:254]: You are right about the station medical ones. Can just revert though. I forgot they were separate on virgo specific. Chomp has them as one. But for the syndicate ones, I could not find a means to separate the selection of the three syndi models. As in, if you were a protector borg, the medi and engi raptor would appear

Comment (Heroman3003) [reply code/modules/mob/living/silicon/robot/sprites/medical.dm:254]: I think I found the cause of that. You have /datum/robot_sprite/dogborg/tall/syndie defined 3 different times with different sprites. You might want to replace /syndie with actual individual module names, like /datum/robot_sprite/dogborg/tall/protector, /datum/robot_sprite/dogborg/tall/mechanist and /datum/robot_sprite/dogborg/tall/combat_medic

Comment (SpaceMan-Jim) [reply code/modules/mob/living/silicon/robot/sprites/medical.dm:254]: Realizing I messed up and am behind on a unGODLY amount of commits. I am just going to close this one and open a fixed on.

**Pull request closed without merging**